frontend/src/errorHandlers/hostErrorHandlers/HostErrorBoundary.spec.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import HostErrorBoundary from './HostErrorBoundary';
+// import AlertModal from '../../components/modals/AlertModal/AlertModal'
+
+jest.mock('../../components/modals/AlertModal/AlertModal.js', () => ({
+  __esModule: true,
+  default: () => <div data-testid="alert-modal">Error Modal</div>,
+}));
+
+jest.mock('../../components/Modal/Modal.js', () => ({
+  __esModule: true,
+  // eslint-disable-next-line react/prop-types
+  default: ({ children }) => <div data-testid="modal">{children}</div>,
+}));
+
+describe('error boundary', () => {
+  let spy;
+
+  // we need to mock the console.error or else jest will error out
+  // https://github.com/facebook/react/issues/11098#issuecomment-523977830
+  beforeEach(() => {
+    spy = jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
I think it's more important to automate the restoration in case any test in the future doesn't need it, and someone forgets to restore it. If this expands (it likely will with error reporting), it's likely every test block should be throwing an error. I'd feel more comfortable leaving it as is.